src/core_plugins/console/public/src/es.js

```diff
@@ -7,6 +7,25 @@ export function getVersion() {
   return esVersion;
 }
 
+export function getContentType(body) {
+  if (!body) return;
+
+  let contentType;
+  try {
+    // there is more than one line
+    const lines = body.split('\n').filter(Boolean);
+    if (!lines.length) throw new Error('not multiline json')
+
+    // each line must be valid json
+    lines.forEach(line => JSON.parse(line));
+    contentType = 'application/x-ndjson';
+  }
+  catch (e) {
+    contentType = 'application/json';
```

Comment:
yes, elasticsearch won't accept content types outside of x-ndjson and json.  it was text/plain before to prevent proxies and load balancers from parsing it, but in practice this also prevents us from getting detailed error messages back from es